Documentation/Proposals/NNNN-make-serialized-trait-api.md

```diff
@@ -0,0 +1,117 @@
+# Make .serialized trait API
+
+* Proposal: [SWT-NNNN](NNNN-make-serialized-trait-api.md)
+* Authors: [Dennis Weissmann](https://github.com/dennisweissmann)
+* Status: **Awaiting review**
+* Implementation: [apple/swift-testing#NNNNN](https://github.com/apple/swift-testing/pull/NNNNN)
+* Review: ([pitch](https://forums.swift.org/t/pitch-make-serialized-trait-public-api/73147))
+
+## Introduction
+
+We propose promoting the existing `.serialized` trait to public API. This trait enables developers to designate tests or test suites to run serially, ensuring sequential execution where necessary.
```

Comment:
Stylistic nitpick: as this file becomes part of the package's documentation, please be sure to wrap at 80 columns as we do for other Markdown files.